### ISSUE TITLE
Extdb2 in 64 bits?

### ISSUE THREAD

**Issue by ghost** [ArmaLife/extDB2#8]
Hello, would it be possible to compile for the 64-bit version?
I keep blocking with "Boost" ...

**Comment by danielstuart14** [comment]
Yeah, it's possible, but it would need some changes to link x64 libs.

You can use extDB 3 commits as a base: 
https://bitbucket.org/torndeco/extdb3/commits/07a3f663756a621c0447d4a93046f8051b5cd664
https://bitbucket.org/torndeco/extdb3/commits/13df139c744940f41fc24e72613d1fc1c32ed598
